packages/prompts/src/index.ts

```diff
@@ -776,10 +778,41 @@ export interface SpinnerResult {
 	readonly isCancelled: boolean;
 }
 
+/**
+ * Global settings for default messages 
+ */
+const promptsSettings = {
+	messages: {
+		cancel: 'Canceled',
+		error: 'Something went wrong',
+	}
+};
```

Comment:
Could this use the [existing global `settings` object](https://github.com/bombshell-dev/clack/blob/46dc0a4bc1b4d8b5ad20aca628334be97887c959/packages/core/src/utils/settings.ts#L10-L22)? We should already expose an `updateSettings()` function